elasticsearch-jmeter/prereqs/prereq.azuredeploy.json

```diff
@@ -1,201 +1,56 @@
-        "jmeterParam": {
-            "yes": " -j ",
-            "no": ""
-        },
-        "cloudAzureParam": {
-            "yes": "[concat(' -po ', variables('cloudAzureStorageAccount'), ' -r ', variables('cloudAzureStorageKey'), ' ')]",
-            "no": ""
-        },
-        "cloudAzureParamValue": "[variables('cloudAzureParam')[variables('cloudAzure')]]",
-        "marvelExportParamValue": "[variables('marvelExportParam')[variables('OS')][variables('marvelCluster')]]",
-        "senseParamValue": "[variables('senseParam')[variables('sense')]]",
-        "marvelParamValue": "[variables('marvelParam')[variables('OS')][variables('marvel')]]",
-        "jmeterParamValue": "[variables('jmeterParam')[variables('jmeterAgent')]]",
-        "afsParamValue": "[variables('afsParam')[variables('OS')][variables('afs')]]",
-        "kibanaTemplates": {
-            "yes": "kibana-resources.json",
-            "no": "empty-resources.json"
-        },
-        "kibanaTemplateFileName": "[variables('kibanaTemplates')[variables('kibana')]]",
+        "dataLBSettings": "[variables('lbBackendPoolsRemoved')]",
         "ubuntuScripts": [
-            "[concat(variables('scriptsLocation'), 'elasticsearch-ubuntu-install.sh', variables('_artifactsLocationSasToken'))]",
-            "[concat(variables('scriptsLocation'), 'kibana-install.sh', variables('_artifactsLocationSasToken'))]",
+            "[concat(variables('scriptsLocation'), 'elasticsearch-ubuntu-install.sh', '')]",
+            "[concat(variables('scriptsLocation'), 'kibana-install.sh', '')]",
             "[concat(variables('sharedScriptsLocation'), 'vm-disk-utils-0.1.sh')]",
             "[concat(variables('sharedScriptsLocation'), 'afs-utils-0.1.sh')]"
         ],
-        "centosScripts": [
-            "[concat(variables('scriptsLocation'), 'elasticsearch-centos-install.sh', variables('_artifactsLocationSasToken'))]",
-            "[concat(variables('scriptsLocation'), 'kibana-centos-install.sh', variables('_artifactsLocationSasToken'))]",
-            "[concat(variables('sharedScriptsLocation'), 'vm-disk-utils-0.1.sh')]"
-        ],
-        "windowsScripts": [
-            "[concat(variables('scriptsLocation'), 'elasticsearch-windows-install.ps1', variables('_artifactsLocationSasToken'))]"
-        ],
         "authSet": {
-            "password": "[variables('adminPassword')]",
-            "sshKey": "[variables('sshPublicKey')]",
-            "type": "[variables('authenticationType')]"
-        },
-        "configurationObject": {
-            "password": {},
-            "sshPublicKey": {
-                "disablePasswordAuthentication": "true",
-                "ssh": {
-                    "publicKeys": [{
-                        "path": "[concat('/home/', variables('adminUsername'), '/.ssh/authorized_keys')]",
-                        "keyData": "[variables('sshPublicKey')]"
-                    }]
-                }
-            }
+            "password": "adminPassword1!",

```

Comment:
Can we use GEN-PASSWORD here?  Also, can you double check that we aren't hard-coding any other passwords in this sample...

Reply:
Sure. I have modified the template and made sure no passwords are hard coded. Thanks.